### PR TITLE
tikv: support security.redact-info-log == marker mode to redact logs.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This pr introduces the newly introduced feature `security.redact-info-log == "marker"` in TiKV.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): ref issue: https://github.com/tikv/tikv/issues/17206, ref pr: https://github.com/tikv/tikv/pull/17212

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -2044,7 +2044,10 @@ Raft Engine 相关的配置项。
 
 ### `redact-info-log` <span class="version-mark">从 v4.0.8 版本开始引入</span>
 
-+ 若开启该选项，日志中的用户数据会以 `?` 代替。
++ 控制是否开启日志脱敏。可选值为 `true` | `"on"`、`false` | `"off"`、`"marker"`。其中，`"marker"` 从 v8.3.0 开始支持。
```
Comment (JaySon-Huang) [inline tikv-configuration-file.md:2047]:
```suggestion
+ 控制是否开启日志脱敏。可选值为 `true` | `"on"`、`false` | `"off"`、`"marker"`。其中，`"on"`、`"off"`、`"marker"` 从 v8.3.0 开始支持。
```

**Comment by JaySon-Huang** [comment]
merge the changes into https://github.com/pingcap/docs-cn/pull/17835

**Comment by LykxSassinator** [comment]
This pr can be closed as it's already merged into https://github.com/pingcap/docs-cn/pull/17835.

**Pull request closed without merging**